libavcodec/qsvenc.c

```diff
@@ -754,6 +752,9 @@ FF_ENABLE_DEPRECATION_WARNINGS
                 q->extco2.MaxQPI = avctx->qmax > 51 ? 51 : avctx->qmax;
                 q->extco2.MaxQPP = q->extco2.MaxQPB = q->extco2.MaxQPI;
             }
+
```

Comment:
This changed the original logic, the original code only need `#if QSV_VERSION_ATLEAST(1, 6)`
This code needs `#if QSV_VERSION_ATLEAST(1, 9)`

Reply:
changed the new code's location.